odoo/addons/base/wizard/base_language_install_views.xml

```diff
@@ -2,34 +2,43 @@
 <odoo>
     <data>
 
+        <record id="view_base_language_switch" model="ir.ui.view">
+            <field name="name">Switch to language</field>
+            <field name="model">base.language.install</field>
+            <field name="arch" type="xml">
+                <form string="Switch to language">
```

Comment:
As you define another form view you have to explicitly set priorities to know which form view is the default one (the one with lower priority is the default one).